app/src/main/java/com/shevchenkovtwo/homework/Movie.kt

```diff
@@ -0,0 +1,21 @@
+package com.shevchenkovtwo.homework
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+data class Movie(
+    @DrawableRes
+    val imageMask: Int? = null,
```

Comment:
What is image mask?

Reply:
Allotted image is imageMask:
![image_mask](https://user-images.githubusercontent.com/53491940/101275108-da84be80-37ab-11eb-8759-a2dc25c57dfe.png)

Reply:
Why it is part of the movie?